docs/DevGuide/BuildOptimization.md

```diff
@@ -130,12 +130,12 @@ the compilation time or memory usage.
 
```

Comment:
I'd like to keep it until I have actually used the split executable approach. I have no experience to agree that this won't be needed anymore. I expect so, but I'm not sure.

Reply:
Ok done